templates/shop/checkout/select_payment/content/form/select_payment/payment/choice/details/paypal.html.twig

```diff
@@ -0,0 +1,5 @@
+{% if hookable_metadata.context.method.gatewayConfig.factoryName == 'sylius.pay_pal' %}
+    <div data-live-ignore>
+        {{ render(controller('Sylius\\PayPalPlugin\\Controller\\PayPalButtonsController::renderPaymentPageButtonsAction', {'orderId': hookable_metadata.context.order.id})) }}
```

Comment:
Can this be replaced by a component?

Reply:
We talked about this, and we decided that we can do id in another task.